x/data-proxy/types/tx.go

```diff
@@ -0,0 +1,31 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
```

Comment:
In case you forget (because I usually forget), we will probably have to add `UnpackInterfaces` for message types that involve `Any` types so we can unwrap proto to the go type.

Reply:
I think with the move to string this is not relevant anymore. :)